db/local_db/status.go

```diff
@@ -36,3 +42,52 @@ func GetStatus() (*RunningDBInstanceInfo, error) {
 
 	return info, nil
 }
+
+// errorIfUnknownService returns an error if it can find a `postmaster.pid` in the `INSTALL_DIR`
```

Comment:
I'd still like a full explanation here - what is that file - why is it an error if the PID in the file is running?